### PR TITLE
Close the game after all players left the game

### LINKED ISSUE #491: [BUG] Close game after players leave
Game is not being closed after all players left the game

### PR DESCRIPTION
**Pull request by tvillegas98**
closes #491 

The main reason we were not closing the game when we should, is because we were not closing the web socket connections for each player. This PR adds that feature in the client and fix a bug in the runner where we were not setting properly the game status after the game finished

### CODE REVIEW

**Review by ncontinanza — DISMISSED** [review]
LGTM! Now you can start a new game that works properly without relaunching the app, nice job.

File: client/Assets/Scripts/LobbyManager.cs [hunk 1]
```diff
@@ -47,10 +50,17 @@ public IEnumerator CreateGame()
 
     public void Back()
     {
+        SocketConnectionManager.Instance.closeConnection();
```
Comment (manucamejo) [inline client/Assets/Scripts/LobbyManager.cs:53]: Is this line duplicated when yo call `BackToLobbyAndCloseConnection`?

Comment (tvillegas98) [reply client/Assets/Scripts/LobbyManager.cs:53]: Yes, indeed is duplicated. Removing it!

Comment (tvillegas98) [reply client/Assets/Scripts/LobbyManager.cs]: Done [71af610](https://github.com/lambdaclass/curse_of_myrra/pull/499/commits/71af610938dd93e771d272918cdd1946596641f0)

**Review by manucamejo — APPROVED** [review]

**Pull request merged**